import_export/resources.py

```diff
@@ -404,13 +400,15 @@ def bulk_create(self, using_transactions, dry_run, raise_errors, batch_size=None
                 else:
                     self._meta.model.objects.bulk_create(self.create_instances, batch_size=batch_size)
         except Exception as e:
-            logger.exception(e)
+            logger.debug(e, exc_info=e)
+            tb_info = traceback.format_exc()
+            result.append_base_error(self.get_error_result_class()(e, tb_info))
```

Comment:
Ran a few tests, indeed memory increase is noticeable. But, unless each bulk operation raises an error, it shouldn't account for more than a few megabytes.  User can now override handle_import_error to prevent this behavior.

As for relying on logger.exception, I think it's an easier interface (and more coherent) to rely on import result. But agree that it can be seen a breaking change